package.json

```diff
@@ -30,5 +30,13 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```

Comment:
![image](https://user-images.githubusercontent.com/31897434/75792536-67f33800-5d6e-11ea-803f-4390f4af1b6e.png)
kindly add the necessary packages.

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/

Reply:
Thanks a lot! I bring the required changes